make_release.py

```diff
@@ -383,6 +383,55 @@ def run_tests():
     echocall('pytest')
 
 
+def update_version(context):
+    if not context['public_release']:
+        return
+
+    chdir(context['build_dir'])
+    version = short(context['release_name'])
+
+    # meta.yaml
+    meta_file = r'condarecipe\larray\meta.yaml'
+    with open(meta_file) as f:
+        lines = f.readlines()
+        lines[2] = "  version: {}\n".format(version)
+        lines[5] = "  git_tag: {}\n".format(version)
+    with open(meta_file, 'w') as f:
+        f.writelines(lines)
+
+    # __init__.py
+    init_file = r'larray\__init__.py'
+    with open(init_file) as f:
+        lines = f.readlines()
+        for i, line in enumerate(lines[:]):
+            if '__version__' in line:
```

Comment:
I would make a generic function for replacing lines in a file and use it for meta, __init__.py and setup.py
```python
def replace_lines(fpath, [(substring_to_find, new_line), (sub2, newline2)]):
    pass
```